app/v2/services/DeleteInsurancePoliciesService.scala

```diff
@@ -40,6 +41,7 @@ class DeleteInsurancePoliciesService @Inject() (connector: DeleteInsurancePolici
       "INVALID_TAX_YEAR"          -> TaxYearFormatError,
       "INVALID_CORRELATIONID"     -> InternalError,
       "NO_DATA_FOUND"             -> NotFoundError,
+      "OUTSIDE_AMENDMENT_WINDOW"  -> RuleOutsideAmendmentWindow,
```

Comment:
Please remove from here and add to the `extraTysErrors` instead so it's clear in the codebase this was introduced for TYS, thanks. This errors map is for Pre TYS i.e. Pre 23-24